tasks/main.yml

```diff
@@ -71,6 +71,7 @@
     comment: '{{ python_deploy__name }}'
     generate_ssh_key: True
     append: True
+    groups: sudo
```

Comment:
Warning od Ansible:
TASK [ansible-python_deploy : Create test user] ********************************
[WARNING]: 'append' is set, but no 'groups' are specified. Use 'groups' for
appending new groups.This will change to an error in Ansible 2.14.
ok: [debian10]